### DISCUSSION TITLE
webdav端口可以单独修改吗，或者web可以单独修改端口吗？

### DISCUSSION THREAD

**Discussion by yyyjk** [AlistGo/alist#3721] [General]
当webdav端口映射到公网后，web页面也同时暴露公网，web就变成了一个未备案的网站。
如果本来就可以单独修改webdav端口的话，还请大佬指出。

**Comment by shoucandanghehe** [comment]
可以用caddy之类的反代webdav

**Reply by M-CU** [reply]
请问caddy2如何反代？我按官方文档进行操作无法反代，大佬能否留个联系方式呢，回头向您请教

**Reply by yyyjk** [reply]
> 可以用caddy之类的反代webdav

谢谢回复，看起来是没法分别改端口了，我也搜索了一遍发现各位都没有改端口的需求。。，反向代理是可以，但是没有直接改端口实在。

**Comment by davidtwchn** [comment]
你是指的 config.json 里面修改端口吗？

**Reply by yyyjk** [reply]
是啊，可以分别定义web和webdav端口吗？

**Reply by davidtwchn** [reply]
噢，只能定义 webdav 端口。

**Reply by davidtwchn** [reply]
我现在反而是在科学上网服务器上，让 caddy 反代 80，443 到 alist 服务上。
